docs/development/adding_tests.md

```diff
@@ -1,18 +1,28 @@
 # Adding tests to TheRock
 
-### Test Flow
+## Test Flow
 
 After TheRock builds its artifacts, we test those artifacts through [`test_packages.yml`](../../.github/workflows/test_packages.yml). The testing flow works as:
 
-`test_sanity_check` -> `configure_test_matrix` -> `test_components`
+`test_sanity_check`--> `configure_test_matrix` --> `test_components`
```

Comment:
````suggestion
```mermaid
graph LR
    test_sanity_check --> configure_test_matrix --> test_components
```
````

(I think, can't check the preview easily right now)